libs/features/footer-promo.js

```diff
@@ -0,0 +1,43 @@
+import { createTag } from '../utils/utils.js';
```

Comment:
Since we already import from utils.js, I think it's better to also import `getConfig` and get the contentRoot that way instead of having to pass it in.